src/app/modules/main/wallet_section/send/view.nim

```diff
@@ -242,3 +240,30 @@ QtObject:
     self.transactionRoutes = newTransactionRoutes()
     self.selectedAssetSymbol = ""
     self.showUnPreferredChains = false
+
+  proc getLayer1NetworkChainId*(self: View): string =
+    return $self.fromNetworksModel.getLayer1Network()
+
+  proc getNetworkColor*(self: View, shortName : string): string =
+    return self.fromNetworksModel.getNetworkColor(shortName)
+
+  proc getNetworkChainId*(self: View, shortName : string): int =
+    return self.fromNetworksModel.getNetworkChainId(shortName)
+
+  proc splitAndFormatAddressPrefix(self: View, text : string, updateInStore: bool): string {.slot.} =
+    return self.delegate.splitAndFormatAddressPrefix(text, updateInStore)
+
+  proc getAddressFromFormattedString(self: View, text : string): string {.slot.} =
+    var address = ""
+    for word in plainText(text).split(':'):
+      if word.startsWith("0x"):
+        address = word
```

Comment:
@Khushboo-dev-cpp add `break` here.
Also in 100% cases the address will be the last element in the array that is the result of `split(':')`, thus it's much better to iterate from the last towards the first element.